lib/tableau/extensions/post_extension/posts/post.ex

```diff
@@ -10,6 +10,12 @@ defmodule Tableau.PostExtension.Posts.Post do
     |> Map.put(:body, body)
     |> Map.put(:file, filename)
     |> Map.put(:layout, Module.concat([attrs.layout || post_config.layout]))
+    |> Map.put_new_lazy(:title, fn ->
+      case Floki.parse_fragment(body) do
```

Comment:
Do you think there is value in just making it use the first h1 even if it's not the first element? 

So like, parse then pipe into Floki.find

Reply:
I was actually thinking of doing it that way initially, but didn't go with that behavior, as it varies across SSGs. Nuxt-content only reads the first element as h1 as the post title, which is the behavior we've got here, while Hugo reads the first h1, regardless of document position.

I'll pop it in because it does seem like a useful thing, although I worry about the edge case of someone writing

```
Hello post

# Some first Heading

```

Where they use H1 (wrongly, according to html semantics, but w/e) as every heading in the document. ¯\\\_(ツ)_/¯ Guess they'll have to not do that